docs/TESTING.md

```diff
@@ -0,0 +1,150 @@
+---
+For release: v1.0
+---
+
+# Test Plan
+
+## Introduction
+
+The idea of this The Test Plan is to communicate the test approach to team members. 
+
+As the team works with the product, they will define the needs for the vNext.
+
+### Team communication
+
+Perform communication through Slack and Github issues.
+
+### Objectives
+
+The 1.0 phase will focus on manual testing for now, and then we'll migrate to automation in the future. Preferably in the next major test plan update, the next app release will follow.
+
+App milestones for 1.0:  
+[https://github.com/WorldHealthOrganization/app/milestone/4](https://github.com/WorldHealthOrganization/app/milestone/4)
+
+### Application architecture
+
+Application is build using Flutter framework. We need to have this in mind and choose appropriate tools and approach.
+
+We're supporting Android and iOS mobile operating systems.
+
+* IOS from 8 to 13
+* Android from 4.1\(API 16\) to 10.0 \(API 29\);
+
+## Plan
+
+### Manual tests
+
+#### Scope
+
+In manual testing, we should focus on all user interface elements, hand washing animation in "Protect Yourself".
+
+#### Examples of configurations to focus on;
+
+* Android GO.
+* Older devices such as Galaxy S3 and iPhone 4s.
+* Smaller screen devices - iPhone 5.
+* Big screens - big tablets such as SM-T670 18.4".
+* Big screen with smaller resolution - iPhone XR.
+* Xiaomi devices.
+
+### Automated checks
+
+We plan to use Firebase Robo test checks for now, and add UI integration tests in the second phase.
+
+We can also use cloud device labs as support in performing various test types.
+
+### Compatibility with Device Hardware
+
+#### Different screen sizes and resolutions;
+
+* The app scales all user interface elements according to current screen density and size.
+* User interface elements do not overlap.
+* Usability or touch issues do not occur. 
+* There is no problematic shrinkage of images because of high DPI/PPI.
```

Comment:
* Minimum color contrast ratios of 3:1 for 18pt text and larger and 4.5:1 of any smaller text.
* Touch targets are at least 44pt on iOS and 48dp on Android (in the smallest dimension)
* All interactive elements are labeled as such with accessibility traits that reflect interactive state and text labels
* Images provide text descriptions
* Screens MUST have meaningful titles.
* The reading order of a screen as conveyed by VoiceOver / TalkBack MUST be logical and intuitive.
* When content is added or deleted from a screen, VoiceOver / TalkBack focus MUST be managed logically and never be lost.
* Fonts should be resizable without breaking layout

Reply:
@avkvirtru Thanks! :)